### PR TITLE
qm-snapshot: add page

### PR DESCRIPTION
**Pull request by byt3h3ad**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

related to #8537

### CODE REVIEW

File: pages/linux/qm-snapshot.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# qm snapshot
+
+> Create a virtual machine snapshot.
```
Comment (navarroaxel) [inline pages/linux/qm-snapshot.md:3]:
```suggestion
> Create virtual machine snapshots.
```

File: pages/linux/qm-snapshot.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Create a snapshot:
```
Comment (navarroaxel) [inline pages/linux/qm-snapshot.md:6]:
```suggestion
- Create a snapshot of a specific virtual machine:
```

File: pages/linux/qm-snapshot.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`qm snapshot {{vm_id}} {{snapshot_name}}`
+
+- Give a textual description or comment:
+
+`qm snapshot {{vm_id}} {{snapshot_name}} --description {{description}}`
+
+- Save the vmstate:
```
Comment (navarroaxel) [inline pages/linux/qm-snapshot.md:14]: Inside the snapshot?
```suggestion
- Create a snapshot including the vmstate:
```
idk what is vmstate actually

File: pages/linux/qm-snapshot.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# qm snapshot
+
+> Create a virtual machine snapshot.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Create a snapshot:
+
+`qm snapshot {{vm_id}} {{snapshot_name}}`
+
+- Give a textual description or comment:
```
Comment (navarroaxel) [inline pages/linux/qm-snapshot.md:10]:
```suggestion
- Create a snapshot with a specific description:
```

File: pages/linux/qm-snapshot.md (hunk 3 above)
Comment (byt3h3ad) [reply pages/linux/qm-snapshot.md:14]: [here's a good explanation](https://wiki.openstack.org/wiki/VMState)

**Review by navarroaxel — APPROVED** [review]

**Comment by byt3h3ad** [comment]
thanks!

**Pull request merged**